phpcs.xml.dist

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0"?>
+<ruleset name="Coding Standards for doctrine">
+    <description>Coding Standards for doctrine.</description>
```

Comment:
This should probably be _"Coding Standard for Doctrine Collections."_ (and same for ruleset name)

Reply:
You're right, I was too lazy to change it (and I probably didn't change on the other projects as well - shame on me!).

Reply:
I've just found out that the `name` attribute and the `description` element are useless here. So I'll just kill them